doc/deprecated.md

```diff
@@ -21,6 +21,7 @@ A list of deprecated features and the version in which they were deprecated foll
 -   Support to NGSI-LD v1.3 in iotagent-node-lib 2.25.0 (finally removed in 2.26.0)
 -   Support groups (provision) statically defined by configuration
 -   Support to in-memory registry (i.e.`deviceRegistry.type=memory`)
+-   Support to legacy expressions 
```

Comment:
Actually, this is not a deprecation case (in the sense it is still supported although not evolved, documented, etc.) but a removal case. Instead of adding the bullet here, maybe it makes sense to incluide it in the table at the end of the document.

Reply:
```suggestion
-   Support to legacy expressions (finally removed in 3.1.0)
```

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1339/commits/cd4a4862ad16300ba95d0dd34e78bc4d47762b30

Reply:
added: 
> (finally removed in 3.2.0)

Because 3.1.0 is still supporting Legacy Expression